src/HTTP/EnvelopeParser.cpp

```diff
@@ -28,12 +28,12 @@ size_t cEnvelopeParser::Parse(const char * a_Data, size_t a_Size)
 	}
 
 	// Start searching 1 char from the end of the already received data, if available:
-	size_t SearchStart = m_IncomingData.size();
-	SearchStart = (SearchStart > 1) ? SearchStart - 1 : 0;
+	auto searchStart = m_IncomingData.size();
```

Comment:
eh. *Java* style names. I mean, it's meant to be verbNoun as in `getThing()` or `resetObject` but it doesn't even *work* for everything. Why is the 'search' part of 'searchStart' so special?

And why did we change style conventions with a couple thousand lines of code already written? :P